Sources/ArcGISToolkit/Common/AttachmentModel.swift

```diff
@@ -81,33 +86,39 @@ internal import os
         }
     }
     
+    deinit {
+        initialLoad?.cancel()
```

Comment:
Probably can use [take()](https://developer.apple.com/documentation/swift/optional/take()) to set it to `nil`.

Reply:
Since I don't believe nulling out the task is required here I'll keep the current approach and merge as is. Thank you for the suggestion, though, it is interesting.